### PR TITLE
cree la app rails usando Postgres.

### PR DESCRIPTION
**Pull request by inayabar**
## Summary 
Cree la app Rails usando Postgres
## Trello Card
https://trello.com/c/9GcK5IHw

### CODE REVIEW

**Comment by epintos** [comment]
@inayabar Please assign to shizus

**Comment by inayabar** [comment]
Esteban, shizus no aparece en la lista de reviewers. Como lo agrego?

2016-12-15 11:15 GMT-03:00 Esteban Pintos <notifications@github.com>:

**Comment by epintos** [comment]
@inayabar Avisale que acepte la invitacion a wolox-training

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  #   log, notice, warning, error, fatal, and panic
+  # Defaults to warning.
+  #min_messages: notice
+
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: entrenamiento_test
```
Comment (agusaldasoro) [inline config/database.yml]: Everything in the App should be in english

File: config/initializers/application_controller_renderer.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Be sure to restart your server when you modify this file.
+
+# ApplicationController.renderer.defaults.merge!(
+#   http_host: 'example.org',
+#   https: false
+# )
```
Comment (agusaldasoro) [inline config/initializers/application_controller_renderer.rb]: Delete unused file

File: config/initializers/backtrace_silencers.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Be sure to restart your server when you modify this file.
+
+# You can add backtrace silencers for libraries that you're using but don't wish to see in your backtraces.
+# Rails.backtrace_cleaner.add_silencer { |line| line =~ /my_noisy_library/ }
+
+# You can also remove all the silencers if you're trying to debug a problem that might stem from framework code.
+# Rails.backtrace_cleaner.remove_silencers!
```
Comment (agusaldasoro) [inline config/initializers/backtrace_silencers.rb]: Delete unused file

File: config/initializers/inflections.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+#   inflect.plural /^(ox)$/i, '\1en'
+#   inflect.singular /^(ox)en/i, '\1'
+#   inflect.irregular 'person', 'people'
+#   inflect.uncountable %w( fish sheep )
+# end
+
+# These inflection rules are supported but not enabled by default:
+# ActiveSupport::Inflector.inflections(:en) do |inflect|
+#   inflect.acronym 'RESTful'
+# end
```
Comment (agusaldasoro) [inline config/initializers/inflections.rb]: Delete unused file

File: config/initializers/mime_types.rb [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# Be sure to restart your server when you modify this file.
+
+# Add new mime types for use in respond_to blocks:
+# Mime::Type.register "text/richtext", :rtf
```
Comment (agusaldasoro) [inline config/initializers/mime_types.rb]: Delete unused file

File: config/locales/en.yml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+#     <%= t('hello') %>
+#
+# To use a different locale, set it with `I18n.locale`:
+#
+#     I18n.locale = :es
+#
+# This would use the information in config/locales/es.yml.
+#
+# To learn more, please read the Rails Internationalization guide
+# available at http://guides.rubyonrails.org/i18n.html.
```
Comment (agusaldasoro) [inline config/locales/en.yml]: Delete all this comments

File: db/seeds.rb [hunk 7]
```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   Character.create(name: 'Luke', movie: movies.first)
```
Comment (agusaldasoro) [inline db/seeds.rb]: Delete unused file

File: app/models/book.rb [hunk 8]
```diff
@@ -0,0 +1,2 @@
+class Book < ApplicationRecord
+end
```
Comment (agusaldasoro) [inline app/models/book.rb]: If this is the kickoff commit, it shouldn't have the Book model

File: app/models/user.rb [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
+         # Hooks
+          before_validation :generate_verification_code, on: :create
+
+  def generate_verification_code
+      self.verification_code = AuthenticableEntity.verification_code
+  end
+end
```
Comment (agusaldasoro) [inline app/models/user.rb]: If this is the kickoff commit, it shouldn't have the user model.

**Pull request merged**